templates/flavors/ipv6/patches/ipv6.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: ["2001:1234:5678:9a40::/58"]
```

Comment:
The cluster CIDR should be different from VNET CIDR block (`2001:1234:5678:`)? Can we use `fc00::/48`?

Reply:
@aramase thoughts on the above?

Reply:
We configure `fc00::/48` for kubenet, but the requirement of pod subnet being part of VNET IP range makes sense for calico.